Make test of getRemoteCertificates() independent of SCTP Transport

dontcallmedom opened the pull request:
Help decoupling tests on DtlsTransport from other mostly-independent features

dontcallmedom committed:
Make test of getRemoteCertificates() independent of SCTP Transport

Help decoupling tests on DtlsTransport from other mostly-independent features

youennf commented on webrtc/RTCDtlsTransport-getRemoteCertificates.html:
```diff
@@ -42,23 +42,13 @@
-      const dtlsTransport1 = sctpTransport1.transport;
-      const dtlsTransport2 = sctpTransport2.transport;
+      const dtlsTransport1 = pc1.getSenders()[0].transport;
+      const dtlsTransport2 = pc1.getReceivers()[0].transport;
```
This should probably be pc2 here.

youennf reviewed (approved):
LGTM with the proposed update.

dontcallmedom committed:
Fix bug

https://github.com/web-platform-tests/wpt/pull/25921#pullrequestreview-500319289
